app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,31 @@
+<section>
```

Comment:
I think we could refactor this html to better match the wireframe, and we can make this work easy for ourselves by using the classes bootstrap provides. Inside your row, I think we should have (in Emmet code, if you copy this into your html and press tab it should expand into what you're after:

(div.col-md-4(div.panel.panel-default>(div.panel-image>img)+(div.panel-body>h4+p)+a.btn.btn-default))*3